Frontend/ui-customer/src/Components/QuoteListModel/View.purs

```diff
@@ -220,39 +221,58 @@ findingRidesView state push =
+  [
+    height WRAP_CONTENT
   , width MATCH_PARENT
   , background Color.white900
-  , visibility if ( null state.quoteListModel ) then GONE else VISIBLE
-  ][ linearLayout[
-    height WRAP_CONTENT
-  , weight 1.0
-  , padding (Padding 16 16 0 16)
-  ][textView
-    [ height WRAP_CONTENT
-    , color Color.black900
-    , textSize FontSize.a_16
-    , fontStyle $ FontStyle.medium LanguageStyle
-    , text case getValueToLocalStore AUTO_SELECTING of
-       "CANCELLED_AUTO_ASSIGN" -> "Select a Ride"
-       "false"                 -> "Select a Ride"
-       _                       -> case (getValueToLocalStore LANGUAGE_KEY) of
-                                    _ -> "Confirming selected ride in" <> " : " <> (fromMaybe configDummy ((filter (\item -> item.id == (fromMaybe "" state.selectedQuote)) state.quoteListModel) !! 0)).timer <> "s"
-                                    -- _ -> "state.timer" <> "s " <> (getString AUTO_ACCEPTING_SELECTED_RIDE) TODO :: NEED TO UPDATE LANGUAGE
-    ]]
-   , linearLayout
-    [ height MATCH_PARENT
-    , width WRAP_CONTENT
-    , gravity CENTER
-    , padding (Padding 16 12 16 12)
-    , onClick push $ const CancelAutoAssigning
-    ][ imageView
-      [ height $ V 24
-      , width $ V 24
-      , imageWithFallback "ny_ic_close,https://assets.juspay.in/nammayatri/images/common/ny_ic_close.png"
-      , visibility if getValueToLocalStore AUTO_SELECTING == "false" || getValueToLocalStore AUTO_SELECTING == "CANCELLED_AUTO_ASSIGN" then GONE else VISIBLE 
-     
+  , visibility if ( null state.quoteListModel ) then GONE else VISIBLE 
+  ][
+    PrestoAnim.animationSet 
+    [ translateInXAnim $ translateInXAnimConfig (getAnimationDuration "") (calculateStartPosition "") (not ( null state.quoteListModel))] $ linearLayout
+    [ height if os == "IOS" then (V 60) else MATCH_PARENT
+    , width MATCH_PARENT
+    , background Color.blue700'
+    , alpha 0.2
+    , visibility if getValueToLocalStore AUTO_SELECTING == "false" || getValueToLocalStore AUTO_SELECTING == "CANCELLED_AUTO_ASSIGN" then GONE else VISIBLE 
+    ][]
+  , linearLayout
+    [ height if os == "IOS" then (V 60) else MATCH_PARENT
+    , width MATCH_PARENT
+    ]
+    [ linearLayout
+    [
+      height MATCH_PARENT
+    , weight 1.0
+    , padding if os == "IOS" then (Padding 16 0 0 0) else (Padding 16 16 0 16)
+    ]
+    [textView
+      [ height MATCH_PARENT
+      , color Color.black900
+      , textSize FontSize.a_16
+      , fontStyle $ FontStyle.medium LanguageStyle
+      , text case getValueToLocalStore AUTO_SELECTING of
+        "CANCELLED_AUTO_ASSIGN" -> getString SELECT_A_RIDE
+        "false"                 -> getString SELECT_A_RIDE
+        _                       -> case (getValueToLocalStore LANGUAGE_KEY) of
+                                      "HI_IN" -> (fromMaybe configDummy ((filter (\item -> item.id == (fromMaybe "" state.selectedQuote)) state.quoteListModel) !! 0)).timer <> "s " <> getString CONFIRMING_SELECTED_RIDE_IN
```

Comment:
It'll be better to have a more descriptive name for `configDummy` that reflects its type, rather than using a generic name.

Reply:
done